Sources/StreamChat/Deprecations.swift

```diff
@@ -5,3 +5,206 @@
+    var currentUserId: UserId { currentUser.id }
+}
+
+public extension NotificationAddedToChannelEvent {
+    @available(*, deprecated, message: "Use channel.cid")
+    var cid: ChannelId { channel.cid }
+}
+
+public extension NotificationRemovedFromChannelEvent {
+    @available(*, deprecated, message: "Use user.id")
+    var currentUserId: UserId { user.id }
+}
+
+public extension NotificationChannelMutesUpdatedEvent {
+    @available(*, deprecated, message: "Use currentUser.id")
+    var userId: UserId { currentUser.id }
+}
+
+public extension NotificationInvitedEvent {
+    @available(*, deprecated, message: "Use member.id")
+    var memberUserId: UserId { member.id }
+}
+
+public extension NotificationInviteAcceptedEvent {
+    @available(*, deprecated, message: "Use member.id")
+    var memberUserId: UserId { member.id }
+}
+
+public extension NotificationInviteRejectedEvent {
+    @available(*, deprecated, message: "Use member.id")
+    var memberUserId: UserId { member.id }
+}
+
+public extension ReactionNewEvent {
+    @available(*, deprecated, message: "Use user.id")
+    var userId: UserId { user.id }
+    
+    @available(*, deprecated, message: "Use message.id")
+    var messageId: UserId { message.id }
+
+    @available(*, deprecated, message: "Use reaction.type")
+    var reactionType: MessageReactionType { reaction.type }
+    
+    @available(*, deprecated, message: "Use reaction.score")
+    var reactionScore: Int { reaction.score }
+}
+
+public extension ReactionUpdatedEvent {
+    @available(*, deprecated, message: "Use user.id")
+    var userId: UserId { user.id }
+    
+    @available(*, deprecated, message: "Use message.id")
+    var messageId: UserId { message.id }
+
+    @available(*, deprecated, message: "Use reaction.type")
+    var reactionType: MessageReactionType { reaction.type }
+    
+    @available(*, deprecated, message: "Use reaction.score")
+    var reactionScore: Int { reaction.score }
+    
+    @available(*, deprecated, message: "Use reaction.updatedAt")
+    var updatedAt: Date { reaction.updatedAt }
+}
+
+public extension ReactionDeletedEvent {
+    @available(*, deprecated, message: "Use user.id")
+    var userId: UserId { user.id }
+    
+    @available(*, deprecated, message: "Use message.id")
+    var messageId: UserId { message.id }
+
+    @available(*, deprecated, message: "Use reaction.type")
+    var reactionType: MessageReactionType { reaction.type }
+    
+    @available(*, deprecated, message: "Use reaction.score")
+    var reactionScore: Int { reaction.score }
+}
+
+public extension TypingEvent {
+    @available(*, deprecated, message: "Use user.id")
+    var userId: UserId { user.id }
+}
+
+@available(*, deprecated, renamed: "NotificationInviteRejectedEvent")
+public typealias NotificationInviteRejected = NotificationInviteRejectedEvent
+
+@available(*, deprecated, renamed: "NotificationInviteAcceptedEvent")
+public typealias NotificationInviteAccepted = NotificationInviteAcceptedEvent
```

Comment:
We need better deprecation message. We need to mention that the whole model is now available

Reply:
✅ Updated deprecation messages